rpm/neutron-opflex-agent.service

```diff
@@ -1,6 +1,8 @@
 [Unit]
 Description=Neutron OpFlex Plugin Agent
-After=syslog.target network.target
+After=syslog.target network.target agent-ovs.service
```

Comment:
Yes, I will submit a pull request for that tonight. I did not have an Ubuntu system to test on yet.



> On Sep 19, 2015, at 4:03 PM, Amit Bose <notifications@github.com> wrote:
> 
> In rpm/neutron-opflex-agent.service:
> 
> > @@ -1,6 +1,8 @@
> >  [Unit]
> >  Description=Neutron OpFlex Plugin Agent
> > -After=syslog.target network.target
> > +After=syslog.target network.target agent-ovs.service
> Do we need something similar for debian? The file to update is debian/neutron-opflex-agent.upstart
> 
> —
> Reply to this email directly or view it on GitHub.
>